css/style.css

```diff
@@ -351,4 +355,92 @@ html {
   height: 11rem;
   border: 0.5rem solid rgba(255, 255, 255, 0.15);
   border-radius: 100%;
+}
+
+/******************************/
+/* MENU SECTION */
+/******************************/
+
+.menu-section {
+  background: url('../gallery/banners/menu-banner.png');
+  background-repeat: repeat;
+  padding-top: 10rem;
+  padding-bottom: 6.4rem;
+  width: 100%;
+  box-shadow: inset 0 0 6rem rgb(0, 0, 0, 0.1);
+  position: relative;
+}
+
+.menu-section:before {
+  content: '';
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to bottom right, rgba(85, 85, 85, 0.2) -100%, transparent 70%);
+  z-index: -1;
+}
+
+.menu-columns {
+  display: grid;
+  grid-template-columns: 43.2rem 43.2rem;
```

Comment:
Maybe it'll be useful to make these columns just of the same size every time and be defined automatically?